.github/workflows/rust.yml

```diff
@@ -18,9 +18,22 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
+        toolchain: [stable]
```

Comment:
I would at least keep beta

Reply:
I personaly think 1.31 and stable are mandatory / must-have because they are _supported_.
beta and nightly are nice-to-have.

Unfortunately, 1.31.1 build failed, so we keep `stable` and `beta` for now.